server/app/routes/v1/grids/grid_certificates.rb

```diff
@@ -0,0 +1,20 @@
+V1::GridsApi.route('grid_certificates') do |r|
+
+  # POST /v1/grids/:name/certificates
+  r.post do
+    data = parse_json_body
+    data[:current_user] = current_user
+    data[:grid] = @grid
+    data[:grid] = @grid
+    outcome = GridCertificates::GetCertificate.run(data)
```

Comment:
I wonder if the `GridCertificates::GetCertificate` should be called something other than `GET`... maybe `GridCertificates::CreateCertificate`, because it seems to do an entirely new CSR each time.

Is there a `GridCertificates::RenewCertificate`?